index.html

```diff
@@ -1126,517 +1231,271 @@ <h4>Time expressions</h4>
-            <tr>
-              <td><code>#time-offset</code></td>
-              <td>permitted</td>
-              <td></td>
-            </tr>
-            <tr>
-              <td><code>#timeContainer</code></td>
-              <td>permitted</td>
-              <td>See [[[#timecontainer-section]]].</td>
-            </tr>
-            <tr>
-              <td><code>#timing</code></td>
-              <td>permitted</td>
-              <td>See [[[#time-expression-section]]].</td>
-            </tr>
-            <tr>
-              <td><code>#transformation</code></td>
-              <td>permitted</td>
-              <td>
-                See constraints at <a href="#profile-constraints">#profile</a>.
-              </td>
-            </tr>
-            <tr>
-              <td><code>#visibility-block</code></td>
-              <td>prohibited</td>
-              <td></td>
-            </tr>
-            <tr>
-              <td><code>#visibility-region</code></td>
-              <td>prohibited</td>
-              <td></td>
-            </tr>
-            <tr>
-              <td><code>#writingMode-horizontal-lr</code></td>
-              <td>prohibited</td>
-              <td></td>
-            </tr>
-            <tr>
-              <td><code>#writingMode-horizontal-rl</code></td>
-              <td>prohibited</td>
-              <td></td>
-            </tr>
-            <tr>
-              <td><code>#writingMode-horizontal</code></td>
-              <td>prohibited</td>
-              <td></td>
-            </tr>
-            <tr>
-              <td><code>#zIndex</code></td>
-              <td>prohibited</td>
-              <td></td>
-            </tr>
-          </tbody>
-        </table>
-      </section> <!-- features -->
 
       <section>
         <h4>Layout</h4>
-        <p>This specification does not put additional constraints on the layout and rendering <a>features</a> defined in [[!ttml-imsc1.2]].</p>
+        <p>This specification does not put additional constraints on the layout and rendering <a>Features</a> defined in [[!ttml-imsc1.2]].</p>
         <div class="note">Layout of the paragraphs may rely on the default TTML region (i.e. if no <code>layout</code> is used in the <code>head</code> element) or may be explicit by the use of the <code>region</code> attribute, if a <code>region</code> element is defined in a <code>layout</code> element in the <code>head</code> element.</div>
       </section>  <!-- Layout -->
 
     </section> <!-- Profile Constraints -->
 
   <section id="conformance">
+    <section id="content-profile-conformance-section">
+      <h3>Conformance of <a>DAPT Scripts</a></h3>
+      <p>Conformant <a>DAPT Scripts</a> are [[ttml2]] <a>Document Instances</a> that
+        conform to the normative provisions of this specification,
+        expressed using the vocabulary of [[ttml2]] in the
+        <dfn data-cite="ttml2#terms-content-profile">content profile</dfn> defined
+        in <a href="#profiles-section"></a>.</p>
```

Comment:
```suggestion
        conform to the normative provisions of this specification.
        Since this specification defines a content profile of [[ttml2]], 
        the profile is also formally defined in the <a href="#profiles-section"></a>
        as required by [[ttml2]].
```

Reply:
Introduce the notion that TTML2 has a formal profile expression language and that the requirements of the spec are expressed in that language.

Also, note that to understand DAPT the syntax and semantics defined in TTML2 apply, but implementing and understanding the profile expression language is not a requirement.